apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/jdbc/SimpleResultSet.java

```diff
@@ -0,0 +1,2467 @@
+/*
+ * Copyright © 2018-2019 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.core.shard.helper.jdbc;
+
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.ObjectOutputStream;
+import java.io.Reader;
+import java.math.BigDecimal;
+import java.net.URL;
+import java.sql.Array;
+import java.sql.Blob;
+import java.sql.Clob;
+import java.sql.Date;
+import java.sql.NClob;
+import java.sql.Ref;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.RowId;
+import java.sql.SQLException;
+import java.sql.SQLWarning;
+import java.sql.SQLXML;
+import java.sql.Statement;
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.sql.Types;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Map;
+
+import com.apollocurrency.aplwallet.apl.core.shard.util.ConversionUtils;
+import org.h2.api.JavaObjectSerializer;
```

Comment:
Maybe remove h2 dependencies?

Reply:
I tried, but those three classes are hard to replace quickly.